### PR TITLE
TICKET-04 User should be able to enter an empty task

### PR DESCRIPTION
**Pull request by alaninfo23**
Adding test to check that the system accept the insert of an empty task.

image

Obs: This branch is depends on TICKET-02, link:
https://github.com/alaninfo23/tasklist-react/pull/4
Please review only the commit:
'Add test case: User should be able to enter an empty task'

### CODE REVIEW

File: src/integration/tests/firstTest.test.jsx [hunk 1]
```diff
@@ -65,6 +65,20 @@ describe("Testes", () => {
...
+    render(<App />);
+    const taskMsg = '';
+
+    const input = screen.getByTestId('INPUT_TASK');
+    userEvent.clear(input);
+    userEvent.type(input, taskMsg);
+    const addTask = screen.getByTestId('ADD_BUTTON');
+    userEvent.click(addTask);
+   
+    expect(screen.queryByTestId(`TASK_CONTAINER_${taskMsg}`)).toBeInTheDocument();
```
Comment (norikoStefany-daitan) [inline src/integration/tests/firstTest.test.jsx:78]:
```suggestion
    expect(screen.queryByTestId(`TASK_CONTAINER_${taskMsg}`)).not.toBeInTheDocument();
```
As the scenario says "User should not be able to enter an empty task"

(I know that there is a bug and it's possible to do it but I think we could leave test failing OR change the scenario to "User should be able to enter an empty task", it's up to you :smile: )

**Review by norikoStefany-daitan — APPROVED** [review]
